packages/discord-bot/package.json

```diff
@@ -9,7 +9,7 @@
     "start": "node ./dist/start.js",
     "build": "yarn generate && tsc -b",
     "dev": "concurrently \"yarn dev-ts\" \"yarn generate --watch\"",
-    "dev-ts": "ts-node-dev --exit-child --respawn -- src/index.ts",
+    "dev-ts": "ts-node-dev --exit-child --respawn -- src/start.ts",
```

Comment:
The build is failing because due to this change, the discord-bot starts up alongside the backend, but both are trying to use port 4000. Running it locally, it appears the discord-bot wins, and backend never actually gets started.

Reply:
hmm we should maybe use a different command name for this then since index.ts doesn't work for running the discord bot locally when dev-ing. Will fix.